manual_content_scan.c

```diff
@@ -1395,6 +1395,9 @@ void manual_content_scan_add_content_to_playlist(
        * > The push function reads our entry as const,
        *   so these casts are safe */
       entry.path      = (char*)playlist_content_path;
+#ifdef HAVE_ENTRYSTATES
+      entry.state     = '\0';
```

Comment:
Change to:

`entry.entry_state     = NULL;`

Reply:
Its unsigned entry.entry_slot now. I think entry.entry_state_slot is a bit to long? Same for the command-line option. The rest is calle entry_state or entry_state_slot

Reply:
Clarification: I keep those two short becaus those  face the user